conda_recipe_manager/fetcher/base_artifact_fetcher.py

```diff
@@ -0,0 +1,57 @@
+"""
+:Description: Provides a base class that all Artifact Fetcher are derived from.
```

Comment:
Nitpicky wording change suggestion
```suggestion
:Description: Provides a base class that all ArtifactFetcher classes are derived from.
```